packages/terminology/src/app.ts

```diff
@@ -0,0 +1,45 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import express from "express";
+import { fhirRouter } from "./router";
+import { initTermServer } from "./sqlite";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+
+dayjs.extend(duration);
+
+async function main() {
+  const app = express();
+
+  app.use(express.json({ limit: "50mb" }));
+  await initTermServer();
```

Comment:
Takes around 10s. It worked fine when branching to staging.

Reply:
It seems pretty close. Check the `configureHealthCheck` on the infra, it seems it will try twice, with 10s between each check. If this takes around 10s, it could stretch to 20 and fail.

The solution might be just to update the healthchecks.

Reply:
I moved the health check to be after the term server init so this shouldnt be an issue